modules/Common/src/EasyAbp.BookingService.Common.Domain/EasyAbp/BookingService/AssetSchedules/AssetScheduleCacheInvalidator.cs

```diff
@@ -0,0 +1,31 @@
+using System.Threading.Tasks;
+using Volo.Abp.Caching;
+using Volo.Abp.DependencyInjection;
+using Volo.Abp.Domain.Entities.Events;
+using Volo.Abp.EventBus;
+using Volo.Abp.MultiTenancy;
+
+namespace EasyAbp.BookingService.AssetSchedules;
+
+public class AssetScheduleCacheInvalidator : ILocalEventHandler<EntityChangedEventData<AssetSchedule>>,
```

Comment:
`EntityChangedEventData` is now inside the UOW, meaning when you remove a cache, someone else tries to rebuild the cache before you commit the transaction, making the cache data wrong.

What we can do are:
1. Always set a short expiration time for caches.
2. Remove the cache after the UOW is complete, or remove it both before and after the UOW is complete. See https://github.com/EasyAbp/EShop/blob/1a05cf4bb156878fd5dfde6678c5e216cae97432/plugins/FlashSales/src/EasyAbp.EShop.Plugins.FlashSales.Application/EasyAbp/EShop/Plugins/FlashSales/FlashSalePlans/FlashSalePlanCacheInvalidator.cs#L24-L32